server/controllers/entities/contributions.js

```diff
@@ -1,17 +1,22 @@
 // An endpoint to list entities edits made by a user
-const patches_ = require('./lib/patches')
+const { byUserId, byDate, byUserIdAndProperty } = require('./lib/patches')
 
 const sanitization = {
   user: { optional: true },
   limit: { default: 100, max: 1000 },
-  offset: { default: 0 }
+  offset: { default: 0 },
+  property: { optional: true },
```

Comment:
the couch view comment `property can be both a label lang or a claim property`, below, could be duplicated (or moved) here, to know early that property could also be label

Reply:
In the current PR implementation, there are 2 possible parameters that get a tailored sanitization: passing a `lang` value as `property` would be rejected, so adding that comment here would add confusion (That comment could be added in the controller function). An alternative implementation could be to regroup those parameters into a single parameter (ex: `filter`), which could be both a property or a lang, but then, unless creating a dedicated sanitization parameter, the sanitization would just validate that its a non-empty string, and property/lang validation would have to be done manually later.

Reply:
Alternative implementation attempted in f0c66ba and af12375, with a unique filter parameter accepting both properties and langs. This would have the advantage to be more easily extended later, for instance, if we were to include filters such as `merge` or `redirection` patches